gradio/examples.py

```diff
@@ -198,7 +237,12 @@ def process_example(self, example_id: int) -> Tuple[List[Any], List[float]]:
             input_component.preprocess(raw_input[i])
             for i, input_component in enumerate(self.inputs)
         ]
-        predictions = self.fn(*processed_input)
+        if inspect.iscoroutinefunction(self.fn):
+            predictions = await self.fn(*processed_input)
+        else:
+            predictions = await anyio.to_thread.run_sync(
+                self.fn, *processed_input, limiter=self.limiter
```

Comment:
self.limiter isn't defined anywhere which will lead to exception when running a blocks api app.

```
  File "/Users/freddy/sources/gradio/gradio/examples.py", line 220, in cache_interface_examples
    prediction = await self.process_example(example_id)
  File "/Users/freddy/sources/gradio/gradio/examples.py", line 244, in process_example
    self.fn, *processed_input, limiter=self.limiter
AttributeError: 'Examples' object has no attribute 'limiter'
```

This is what I get running `blocks_inputs`

Reply:
This is because limiter is defined on Blocks. Btw since this is just examples, we can remove limiter support from it, not need IMO.

```suggestion
                self.fn, *processed_input
```

Reply:
Agreed!

Reply:
Thanks, will remove